source/ConfigMgrCBDsc.psd1

```diff
@@ -64,7 +65,7 @@
 
             # Tags applied to this module. These help with module discovery in online galleries.
             Tags = @('DesiredStateConfiguration', 'DSC', 'DSCResourceKit', 'DSCResource', 'ConfigMgrCBDsc','CMAccounts','CMIniFile','Collections',
-            'CMBoundaries','CMForestDiscovery','ClientStatusSettings','BoundaryGroups','ManagementPoint','CMAssetIntelligencePoint')
+            'CMBoundaries','CMForestDiscovery','ClientStatusSettings','BoundaryGroups','ManagementPoint','CMAssetIntelligencePoint','CMSoftwareUpdatePoint')
```

Comment:
in another PR we are removing the CM precursor from the other settings except for CMIniFile, remove CM from CMSoftwareUpdatePoint